File: leetcode/2185/c/main.c
int prefixCount(char** words, int wordsSize, char* pref) {
    int cnt = 0;
    int pref_size = strlen(pref);
    for (int i = 0; i < wordsSize; i++) {
        int found = true;
        char* word = words[i];
        int word_size = strlen(word);
        if (word_size < pref_size) { continue; }
        for (int j = 0; j < pref_size; j++) {
            if (word[j] != pref[j]) {
                found = false;
                break;
            }
        }
        if (found) {
            cnt++;
        }
    }

    return cnt;
}

File: leetcode/2185/rs/src/main.rs
struct Solution;
impl Solution {
    pub fn prefix_count(words: Vec<String>, pref: String) -> i32 {
        words.iter().filter(|&word| word.starts_with(&pref)).count() as i32
    }
}
fn main() {
    println!("Hello, world!");
}
